Assets/GothicVR/Scripts/UI/TurnSettingDropdownController.cs

```diff
@@ -0,0 +1,52 @@
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+using UnityEngine.XR.Interaction.Toolkit;
+using GVR.Phoenix.Interface;
+public class TurnSettingDropdownController : MonoBehaviour
+{
+    public GameObject textAsset;
+    public GameObject locomotionsystem;
+    public ActionBasedSnapTurnProvider snapTurn;
+    public ActionBasedContinuousTurnProvider continuousTurn;
+    // Start is called before the first frame update
+    void Awake()
```

Comment:
Still a wrong comment. ;-)

```suggestion

    void Awake()
```